### ISSUE TITLE
HTML files in Google Keep Importer labelled skipped

### ISSUE THREAD

**Issue by daledesilva** [obsidianmd/obsidian-importer#57]
In the final merge of the Google Keep import format, files found in the zip which are irrelevant to the import have been set to report as "Skipped". I would recommend against this.

As I found with my previous import plugin, mentioning these files as being skipped causes the user to to think something has gone wrong when it hasn't. And it can also can cause them to create GitHub issues about it unnecessarily.

For instance, HTML files are ignored in the import because they only duplicate the JSON files which we do import.
Reporting them as skipped, therefore, implies an issue where there is none, and bloats the output log such that the user finds it harder to notice the actual relevant messages such as notes that have been entirely skipped because they had an "archived property".

Note, this only affects importing as a zip because when selecting files directly you aren't given the ability to select unimportable files.

**Comment by lishid** [comment]
Are there anything else apart from HTML files? There wasn't anything documented in the original PR so I thought this must have been an oversight. I can add a check and put a comment in, but I'd like to be clear what exact scenario we are expecting.

**Comment by daledesilva** [comment]
- Html duplicates
- Another html summary
- A text file with labels summary

**Comment by lishid** [comment]
Ok, do you think it's best to ignore all unsupported files or ignore specific extensions?

**Comment by NomarCub** [comment]
I second this. I just tried the plugin, when I saw the Keep integration was done.
This message is really not useful. I can't eve copy the logs to disect them elsewhere.
![image](https://github.com/obsidianmd/obsidian-importer/assets/5298006/6ce5f322-7295-47b5-bf20-3e11b9b5ea11)

To be cautious about ignoring, I would make it so only HTML files with exact JSON name matches, and the specific currently known summary files get ignored, so future additions to the Keep export format would be noticed.
